client/src/main.tsx

```diff
@@ -32,7 +32,7 @@ const themeConfig: ThemeConfig = {
       colorPrimary: '#1cb500',
     },
     Menu: {
-      itemHeight: 60,
+      itemHeight: 30,
```

Comment:
This may also affect the data depot. Can we confirm this only affects the workspace menu?

Reply:
Yes, I did check around - the only antd Menu in the repo is here in the workspace for apps that I can tell.